subjects/reversesecondhalf/README.md

```diff
@@ -0,0 +1,42 @@
+## reversesecondhalf
+
+### Instructions
+Write a function that takes a `string` as an argument and prints the second half reversed. If the length of the string is odd, round it up.
```

Comment:
It is only a suggestion, instead of "...and prints the second half reversed."  could be "...and returns the second half reversed."

Reply:
I think it's ok as it is